lettuce/cli.py

```diff
@@ -64,6 +64,7 @@ def benchmark(ctx, steps, resolution, profile_out):
     collision = BGKCollision(lattice, tau=flow.units.relaxation_parameter_lu)
     streaming = StandardStreaming(lattice)
     simulation = Simulation(flow=flow, lattice=lattice,  collision=collision, streaming=streaming)
+    simulation.reporters.append(VTKReporter(lattice, flow, filename="Data", interval=1))
```

Comment:
The benchmark should run without creating any files (benchmarking should be done without any overhead for file I/O and analysis).

Could you write a unit test instead? - for a very coarse resolution and in 1D, 2D, and 3D?

Reply:
wip